sorc/CMakeLists.txt

```diff
@@ -35,10 +35,10 @@ endif()
 
 # Build UFS weather model
 if (BUILD_UFS)
-  list(APPEND TARGET_LIST ufs-weather-model)
+  list(APPEND TARGET_LIST UFS)
```

Comment:
@MichaelLueken This needs to be removved because it will re-introduce the bug with building without installing the ufs-weather-model.